source/gfoidl.Base64/Internal/Base64Encoder/Base64Encoder.Decode.cs

```diff
@@ -406,7 +406,6 @@ private static bool Avx2Decode<T>(ref T src, ref byte destBytes, int sourceLengt
             if (success)
```

Comment:
This is only for debug, so `success` could be removed, the event fired in every iteration, thus cleaning up the code (less `#if`s).